app.js

```diff
@@ -15,12 +16,16 @@ app.get("/api", getAll);
 app.get("/api/articles/:article_id", getArticleById);
 app.get("/api/articles", getArticles);
 app.get("/api/articles/:article_id/comments", getCommentsByArticleId);
+app.post("/api/articles/:article_id/comments", addCommentByArticleId);
 
 app.use((err, req, res, next) => {
   switch (err.code) {
     case "22P02":
       res.status(400).send({ msg: "Bad request" });
       break;
+    case "23503":
+      res.status(400).send({ msg: "Empty user!" });
+      break;
   }
   next(err);
 });
```

Comment:
Lovely use case for a swithc statement! I think I'd like to see your error handling middleware in a new file now :)